doc/FAQ

```diff
@@ -111,7 +111,8 @@
    you may encounter. You may also try digging in our mailing list
    archive: many problems have been solved there.
    If you still can't get it to work please either subscribe to the KVIrc
-   mailing list (see www.kvirc.net) and report the problem there or join #kvirc
+   mailing list (see www.kvirc.net) and report the problem there
```

Comment:
This file is a mess, I need to check it properly, thers more than just a missing . :)